components/Section/Section.tsx

```diff
@@ -0,0 +1,33 @@
+import { ReactNode } from 'react'
+
+interface Props {
+  className?: string
+  p?: string
+  px?: string
```

Comment:
Here, a parameter in the form of qweqwe should not be given. Please strengthen validation.

example, this is invalid:

```tsx
<Section px="askdasş" />
```

example, this is valid:

```tsx
<Section px="1rem" />
```

Please make a type like this:

```ts
type Unit = '%' | 'px' | 'em' | 'vh' | 'vh'

type UnitProp = `${number}${Unit}`

type Props = {
  px: UnitProp
}
```

Reply:
Should i define the `Unit` and `UnitProp` types as a local or export them from `types` folder?

Reply:
It would be better to export from types folder